vertical-pod-autoscaler/pkg/utils/vpa/capping.go

```diff
@@ -30,88 +30,127 @@ func NewCappingRecommendationProcessor() RecommendationProcessor {
 	return &cappingRecommendationProcessor{}
 }
 
+type cappingAction = string
+
+var (
+	cappedToMinAllowed cappingAction = "capped to min allowed"
```

Comment:
capped to min allowed from VPA resourcePolicy? or VPA containerPolicy
min allowed seems a bit too generic

Reply:
I was wondering if I see minAllowed I'll know immediately that this is the minAllowed from the VPA object. But I might be paranoid.

Reply:
ok, changed minAllowed/maxAllowed